packages/aws-rfdk/lib/lambdas/nodejs/x509-certificate/acm-handlers.ts

```diff
@@ -234,9 +251,9 @@ export class AcmCertificateImporter extends DynamoBackedCustomResource {
 /* istanbul ignore next */
 export async function importCert(event: CfnRequestEvent, context: LambdaContext): Promise<string> {
   const handler = new AcmCertificateImporter(
-    new ACM({ apiVersion: ACM_VERSION }),
```

Comment:
Why'd we remove these? Are they no longer supported? Just wondering what the impact would be of this change

Reply:
Yes, `apiVersion` is no longer supported.  I'm having trouble finding a definitive reference for this, but see for example this ticket: [Remove apiVersion from configuration](https://github.com/aws/aws-sdk-js-v3/issues/1869).

Also, I ran the code through the [aws-sdk-js-codemod](https://github.com/aws/aws-sdk-js-codemod) tool, and it added annotations such as:

```
// The key apiVersion is no longer supported in v3, and can be removed.
// @deprecated The client uses the "latest" apiVersion.
apiVersion: '2014-11-06',
```